transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/auth/AuthHandlerImpl.kt

```diff
@@ -125,6 +140,10 @@ internal class AuthHandlerImpl(
         vault.authRefreshToken = NO_REFRESH_TOKEN
     }
 
+    override fun hasRefreshToken(): Boolean {
```

Comment:
I think we already have a private (extension) function with the same name. Can we check if it makes sense to somehow reuse it. It might be confusing to have 2 functions with the same name inside same class.